### PR TITLE
Feat/loader and profile

### PR DESCRIPTION
**Pull request by manish591**
#### Added Profile Page And Loader To Give User Feedback
* Added profile page, showing user details and a logout button
* Added Not-found page
* Added toast using react-hot-toast package

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -10,22 +12,36 @@ import {
   Playlist,
   Home,
   VideoPage,
+  UserProfile,
```
Comment (varnita619) [inline src/App.js:15]: Good use of barrel exports

File: src/hooks/useAppServices.js [hunk 2]
```diff
@@ -222,9 +245,11 @@ const useAppServices = () => {
           type: ACTION_TYPES.GET_PLAYLIST_DATA,
           payload: { myPlaylist: arr },
         });
+        toast("Video Added To Playlist");
       }
     } catch (err) {
       console.log(err);
+      toast.error("Unable To Add Video To The Playlist! Try Again Later");
```
Comment (varnita619) [inline src/hooks/useAppServices.js:252]: Proper toast messages

**Review by varnita619 — APPROVED** [review]
Code looks clean
With good namin conventions
Ready to merge

File: src/pages/Home/Home.jsx [hunk 3]
```diff
@@ -4,10 +4,14 @@ import "./Home.css";
 import food from "../../assets/photo-1475090169767-40ed8d18f67d-removebg-preview.png";
 import { Link } from "react-router-dom";
 import axios from "axios";
-import { useStateContext } from "../../hooks";
 import { ACTION_TYPES } from "../../reducer";
 import { CategoryCard } from "../../components";
-import { useAuthContext, useAuth } from "../../hooks";
+import {
+  useAuthContext,
```
Comment (Devansu-Yadav) [inline src/pages/Home/Home.jsx:10]: Great use of barrel imports.

**Review by Devansu-Yadav — APPROVED** [review]
Like:-
- Clean, modular, and readable code.
- Great use of barrel imports.
- UI looks very clean.
- Really great thinking behind the UI design for mobile screens.

Suggestions:
- Update favicon for the app
- Can add a link to the Landing page for the App logo on NavBar ([Link](https://deploy-preview-15--foodio-watch.netlify.app/explore)).

File: src/pages/Home/Home.jsx (hunk 3 above)
Comment (manish591) [reply src/pages/Home/Home.jsx:10]: Thanks for suggestion, will implement those.

**Pull request merged**